docker/softlinkstatic.py

```diff
@@ -7,10 +7,11 @@
 with open(STATIC) as static_fp:
     static_files = json.load(static_fp)
 
-for path in static_files['paths']:
-    full_path = os.path.join('./static', path)
-
-    try:
-        os.unlink(full_path)
-    except OSError:
-        pass
+for orig, hashed in static_files['paths'].items():
+    if '?' in orig:
+        continue
```

Comment:
would be nice to catch these too, but I get ya. Could just split on `?`, but my other idea for this doesn't see those query params, so we can fix that way.